app/controllers/assignment_invitations_controller.rb

```diff
@@ -35,6 +48,24 @@ def check_user_not_previous_acceptee
     redirect_to success_assignment_invitation_path
   end
 
+  def check_authorized_repo_setup
```

Comment:
I prefer `ensure` over `check` in these types of methods. Ensure sort of implies something will be done if the check fails.